inginious/frontend/pages/course_admin/aggregation_task.py

```diff
@@ -38,7 +39,23 @@ def page(self, course, aggregationid, task):
                                                     "archive": False,
                                                     "input": False}).sort([("submitted_on", pymongo.DESCENDING)]))
         data = [dict(list(f.items()) + [("url", self.submission_url_generator(str(f["_id"])))]) for f in data]
+
         if "csv" in web.input():
             return make_csv(data)
-
-        return self.template_helper.get_renderer().course_admin.aggregation_task(course, aggregation, task, data)
+            
+        # Get best submissions (submission for evaluation)
+        # Need this for statistics computation
+        # Not really optimized for now...
+        user_tasks = list(self.database.user_tasks.find({"username": {"$in": aggregation["students"]}, 
+                                                       "courseid": course.get_id(),
+                                                       "taskid": task.get_id()},
+                                                       {"submissionid": 1, "_id": 0}))
+        best_submissions_list = [] # list containing Ids of best submissions
```

Comment:
``best_submissions_list = [ut["submissionid"] for ut in user_tasks]`` would be better processed by Python. However, I'm wondering if only one field is required if mongo function ``distinct`` would not do the job better, as submissionids are unique.